### PR TITLE
adding database server examples

### PR DESCRIPTION
**Pull request by robertsilen**
Adding database server examples that use the MySQL protocol (MariaDB, MySQL, Percona, TiDB) as per discussion https://github.com/php/doc-en/issues/3628

### CODE REVIEW

File: reference/mysqlinfo/concepts.xml [hunk 1]
```diff
@@ -186,5 +186,20 @@ $pdo = new PDO("mysql:host=localhost;dbname=world;charset=utf8mb4", 'my_user', '
...
+ 
+ <section xml:id="mysqlinfo.concepts.databases">
+  <title>Examples of compatible database servers</title>
+
+  <para>
+  The MySQL extensions support the MySQL protocol. 
+  Examples of compatible database servers are 
+  <link xlink:href="https://mariadb.org/">MariaDB Server</link>, 
+  <link xlink:href="https://www.percona.com/mysql/software/percona-server-for-mysql">MySQL Server</link>, 
+  <link xlink:href="">Percona Server for MySQL</link>,
```
Comment (cmb69) [inline reference/mysqlinfo/concepts.xml:198]: Either add an URL here, or don't mark up as link.

File: reference/mysqlinfo/concepts.xml [hunk 2]
```diff
@@ -186,5 +186,20 @@ $pdo = new PDO("mysql:host=localhost;dbname=world;charset=utf8mb4", 'my_user', '
   </example>
 
  </section>
+ 
+ <section xml:id="mysqlinfo.concepts.databases">
+  <title>Examples of compatible database servers</title>
+
+  <para>
+  The MySQL extensions support the MySQL protocol.
```
Comment (cmb69) [inline reference/mysqlinfo/concepts.xml:194]: The indentation should be a single space (also below).

**Review by cmb69 — CHANGES_REQUESTED** [review]
Thank you for the PR! This looks generally good to me, also some issues still need to be resolved.

@kamil-tekiela, what do you think?

File: reference/mysqlinfo/concepts.xml (hunk 1 above)
Comment (robertsilen) [reply reference/mysqlinfo/concepts.xml]: fixed MySQ and Percona URLs in link tags

File: reference/mysqlinfo/concepts.xml (hunk 2 above)
Comment (robertsilen) [reply reference/mysqlinfo/concepts.xml]: fixed

File: reference/mysqlinfo/concepts.xml [hunk 3]
```diff
@@ -186,5 +186,20 @@ $pdo = new PDO("mysql:host=localhost;dbname=world;charset=utf8mb4", 'my_user', '
...
+  <title>Examples of compatible database servers</title>
+
+  <para>
+  The MySQL extensions support the MySQL protocol. 
+  Examples of compatible database servers are 
+  <link xlink:href="https://mariadb.org/">MariaDB Server</link>, 
+  <link xlink:href="https://www.percona.com/mysql/software/percona-server-for-mysql">MySQL Server</link>, 
+  <link xlink:href="">Percona Server for MySQL</link>, 
+  and 
+  <link xlink:href="https://www.pingcap.com/tidb/">TiDB</link>.
```
[the remark this answers is not in the document]
Comment (robertsilen) [reply reference/mysqlinfo/concepts.xml]: fixed

File: reference/mysqlinfo/concepts.xml [hunk 4]
```diff
@@ -186,5 +186,20 @@ $pdo = new PDO("mysql:host=localhost;dbname=world;charset=utf8mb4", 'my_user', '
...
 
  </section>
+ 
+ <section xml:id="mysqlinfo.concepts.databases">
+  <title>Examples of compatible database servers</title>
+
+  <para>
+  The MySQL extensions support the MySQL protocol. 
+  Examples of compatible database servers are 
+  <link xlink:href="https://mariadb.org/">MariaDB Server</link>,
```
[the remark this answers is not in the document]
Comment (robertsilen) [reply reference/mysqlinfo/concepts.xml]: Added URLs now here, will change to variables when this is merged: https://github.com/php/doc-en/pull/3639

**Comment by cmb69** [comment]
Hmm, CI is reporting "Namespace prefix xlink for href on link is not defined"; not sure why, since apparently the respective XML namespace is declared.

**Review by kamil-tekiela — APPROVED** [review]
It looks fine to me but I wouldn't duplicate the information. Pick one spot it fits the best.

**Comment by cmb69** [comment]
The build still fails, likely because of

https://github.com/php/doc-en/blob/227766a18bcf610c39167c4c59a5e8a8a0b504b7/reference/mysqlinfo/set.xml#L4

The XML namespace declaration should probably be added to this element.

**Review by cmb69 — COMMENTED** [review]
This looks good to me now, but I think we should merge https://github.com/php/doc-base/pull/144 first, and then adapt and merge this PR (might be a bit easier to handle for translators).

**Review by cmb69 — APPROVED** [review]

**Comment by cmb69** [comment]
Since https://github.com/php/doc-base/pull/144 has been merged, could you please update the PR to use these entities?

**Comment by cmb69** [comment]
Thank you very much!

**Pull request merged**